File: input-files/math_utils.py
def add(a: int, b: int) -> int:
    return a + b

def subtract(a: int, b: int) -> int:
    return a - b

def multiply(a: int, b: int) -> int:
    return a * b

class Calculator:
    def add(self, a: int, b: int) -> int:
        return add(a, b)

    def subtract(self, a: int, b: int) -> int:
        return subtract(a, b)

    def multiply(self, a: int, b: int) -> int:
        return multiply(a, b)

File: input-files/math_utils.ts
export function add(a: number, b: number): number {
    return a + b;
}

export function subtract(a: number, b: number): number {
    return a - b;
}

export function multiply(a: number, b: number): number {
    return a * b;
}

export class Calculator {
    add(a: number, b: number): number {
        return add(a, b);
    }

    subtract(a: number, b: number): number {
        return subtract(a, b);
    }

    multiply(a: number, b: number): number {
        return multiply(a, b);
    }
}